.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+### Description
+
+<!-- What has changed and why has it changed -->
+
+#### Remaining Tasks
+
+<!-- Tasks left to do, either in this MR or as future work -->
+
+#### Related MRs in other projects
+
+<!-- Links of other related MR to this one -->
+
+#### Additional merge criteria
+
+<!-- Here, add any extra criteria you consider mandatory for merging on top of the usual criteria -->
+
+#### Breaking changes
+
+<!-- Here, add all the breaking changes that this MR involves if there is any -->
+
+### Checklist
+
+- [ ] Title of the MR follows the right format (see the internal documentation).
```

Comment:
Given this is a public repo, maybe provide instruction for the format somewhere here? See internal documentation not very helpful

Reply:
Maybe having the document in accessible to public ? Isn't something that need to be internal right ?